public/app/plugins/panel/table/models.gen.ts

```diff
@@ -32,7 +32,14 @@ export const defaultPanelOptions: PanelOptions = {
 };
 
 export const defaultPanelFieldConfig: TableFieldOptions = {
-  displayMode: TableCellDisplayMode.Auto,
+  cellOptions: {
+    displayMode: TableCellDisplayMode.Auto,
+  },
   align: 'auto',
   inspect: false,
 };
+
+export interface TableCellEditorProps {
+  cellOptions?: TableCellOptions;
+  onCellOptionsChange: Function;
+}
```

Comment:
```suggestion
export interface TableCellEditorProps<T> {
  cellOptions?: T;
  onChange: (value: T) => void;
}
```

Not sure this type should be in the auto generated file